hoomd/md/module-md.cc

```diff
@@ -279,13 +295,13 @@ PYBIND11_MODULE(_md, m)
     m.def("make_wall_field_params", &make_wall_field_params);
     export_PotentialExternal<PotentialExternalPeriodic>(m, "PotentialExternalPeriodic");
     export_PotentialExternal<PotentialExternalElectricField>(m, "PotentialExternalElectricField");
-    export_PotentialExternalWall<EvaluatorPairLJ>(m, "WallsPotentialLJ");
-    export_PotentialExternalWall<EvaluatorPairYukawa>(m, "WallsPotentialYukawa");
-    export_PotentialExternalWall<EvaluatorPairSLJ>(m, "WallsPotentialSLJ");
-    export_PotentialExternalWall<EvaluatorPairForceShiftedLJ>(m, "WallsPotentialForceShiftedLJ");
-    export_PotentialExternalWall<EvaluatorPairMie>(m, "WallsPotentialMie");
-    export_PotentialExternalWall<EvaluatorPairGauss>(m, "WallsPotentialGauss");
-    export_PotentialExternalWall<EvaluatorPairMorse>(m, "WallsPotentialMorse");
+    // export_PotentialExternalWall<EvaluatorPairLJ>(m, "WallsPotentialLJ");
```

Comment:
```suggestion
    // TODO: Port walls to HOOMD v3
    // export_PotentialExternalWall<EvaluatorPairLJ>(m, "WallsPotentialLJ");
```
Let's document why we are commenting this out.